### DISCUSSION TITLE
relocate Sumatra's default working file

### DISCUSSION THREAD

**Discussion by kelvin-koolohm** [sumatrapdfreader/sumatrapdf#4268] [Q&A]
When I installed Sumatra, I simply went along with whatever it asked for. Somehow, this included using
my "Documents" folder for all of its archives. I would now like to create a subfolder in Documents called
"Sumatra-archives" for it to use for that purpose. Is there some way (easy) to create that folder, hand transfer
all the pertinent PDF files in "Documents" (some PDF files there don't belong to Sumatra) to that new
subfolder, and get Sumatra to use that subfolder as its default?
I realize that there is a file that contains the locations of all the files listed on the top line of Sumatra and I am
prepared to do a search and replace on the data in that file but I need to know if all that would be sufficient?

**Comment by kjk** [comment]
Sumatra doesn't store data in `Documents` folder. See https://www.sumatrapdfreader.org/docs/How-we-store-settings

You can change data directory with cmd-line arg `-appdata`. See https://www.sumatrapdfreader.org/docs/Command-line-arguments

If you copy existing data to that directory everything should work fine.
